src/test/java/com/salesforce/dynamodbv2/mt/util/CachingAmazonDynamoDbStreamsTest.java

```diff
@@ -1345,6 +1340,13 @@ void testGetRecordsLockTimeout() throws InterruptedException {
         assertCacheMisses(streams, 0, 0);
     }
 
+    private Striped<Lock> getMockStripedLock(Lock lock) {
```

Comment:
How about calling it `mockStripedLock` (since it's creating something vs just getting something) and moving it where the other `mock*` methods are defined?